### PR TITLE
Customize this repo to host the Racklet website.

### PR DESCRIPTION
**Pull request by luxas**
Tried this out, let's see if this works :smile:. Will over time make sure that this site is not too similar to the Flux one, when we have some more graphics, content, blogs and color palette. But for now, I'm re-using the base layout from Flux. Thanks to the Flux maintainers for a great starting point :heart:

@twelho @chiplet

### CODE REVIEW

File: README.md
Comment (twelho) [inline README.md]:
```suggestion
This repo houses the assets used to build the Racklet project's landing page at https://racklet.io.
```

File: README.md
Comment (twelho) [inline README.md]:
```suggestion
> **Note**: The sources for an individual Racklet project's (e.g. `racklet/racklet`) documentation at https://docs.racklet.io are housed in its respective Racklet repo (e.g. https://github.com/racklet/racklet). Issues and pull requests for a specific project's documentation should be registered in its own repo.
```
I think this is what you wanted to say?

File: README.md
Comment (twelho) [inline README.md]: Maybe mention the fluxcd/website maintainers by name and GH handle to not remove attribution?

File: config.toml
Comment (twelho) [inline config.toml:85]: At some point we probably want to create a separate account for the project like fluxcd has.

File: config.toml
Comment (twelho) [inline config.toml]:
```suggestion
subtitle = "Open Source scale model of Data Centers using commodity compute like Raspberry Pis."
```

File: config.toml
Comment (twelho) [inline config.toml]:
```suggestion
Racklet is a fully-integrated, miniature server rack. Racklet aims to inspire users to explore how modern,
```

File: config.toml
Comment (twelho) [inline config.toml]:
```suggestion
We aspire to improve the status quo of "secure-by-default" solutions and concepts available by using
```

File: config.toml
Comment (twelho) [inline config.toml]:
```suggestion
manner. For non-off-the-shelf casing and PCBs we provide 3D-printable designs and easily reproducible
```

File: config.toml
Comment (twelho) [inline config.toml]:
```suggestion
schematics respectively.
```

File: config.toml
Comment (twelho) [inline config.toml]:
```suggestion
rest of the rack or disregarding existing, functioning parts. This will minimize E-waste produced over time.
```

File: config.toml
Comment (twelho) [inline config.toml]:
```suggestion
modules implementing the same interfaces, or extensibility where the user demands customized features.
```
"custom" is an informal abbreviation, it's better to always write it out.

File: config.toml
Comment (twelho) [inline config.toml]:
```suggestion
The Racklet project aspires to engage a diverse set of people in the open source world to work
```

File: config.toml
Comment (twelho) [inline config.toml]:
```suggestion
We'd love to see you collaborate with us on this project! No contribution is too small, we value all input and perspectives.
```

**Review by twelho — APPROVED** [review]
Thanks @luxas, LGTM, let's see if this works out. I've suggested a couple of grammar improvements, also we should probably add the names and GH handles of the original authors of fluxcd/website into `README.md` to avoid removing original attribution.

File: config.toml
Comment (luxas) [reply config.toml]: I think it should be "collaborate to"?

Comment (twelho) [reply config.toml]: Typo on my side, should be "on", AFAIK it's "contribute to sth" and "collaborate on sth". Collaboration implies mutual communication, thus "on" to point out participation in the project/community (unlike contributing to something which can be a one-off external action). For clarity though the sentence could be written out in full:
```
We'd love to see you collaborate with us on this project!
```

Comment (twelho) [reply config.toml]: Suggestion changed to reflect that now.

File: README.md
Comment (twelho) [inline README.md]:
```suggestion
Instead of reinventing the wheel, we figured we'd use the same website template over time (Racklet is also Apache 2.0-licensed). As time passes, we expect to diverge more and more from the original website template (intentionally) to not confuse readers with two website UIs that look the same.
```

File: README.md
Comment (twelho) [inline README.md]:
```suggestion
Thanks a lot to the maintainers of the `fluxcd/website` project, listed here for attribution:
```

File: README.md
Comment (twelho) [inline README.md]:
```suggestion
Daniel Holbach, Weaveworks <daniel@weave.works> (GitHub: @dholbach, Slack: dholbach)
```

File: README.md
Comment (twelho) [inline README.md]:
```suggestion
Hidde Beydals, Weaveworks <hidde@weave.works> (GitHub: @hiddeco, Slack: hidde)
```

File: README.md
Comment (twelho) [inline README.md]:
```suggestion
Scott Rigby, Weaveworks <scott@weave.works> (GitHub: @scottrigby, Slack: scottrigby)
```

**Review by twelho — APPROVED** [review]
Some more typo/formatting fixes, looking good :+1:

**Pull request merged**
